lib/camping/gear/filters.rb

```diff
@@ -0,0 +1,48 @@
+# This gear is originally from filtering_camping gem by judofyr, and techarc.
+module Gear
+	module Filters
```

Comment:
Could you please fix indenting before merging? It's hard to read.

Reply:
This looks great, easy to follow what this class does.

Reply:
Changed this file back to spaces. We'll stick with that for now.